doc/RecordingControl.xml

```diff
@@ -1684,8 +1787,420 @@ SetRecordingJobMode(JobToken, Active)
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="media/RecordingControl/cloudRecordingLogicalModel.svg" contentwidth="79mm" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+    </section>
+    <section xml:id="_refCloudRecordingSpans">
+      <title>Spans</title>
+      <para>
+        When a recording is activated, the device shall open new spans for this recording according to the rules of the configured <literal>Format</literal>.
+        Inversely, when a recording is deactivated, the device shall close all open spans of this recording.
+        When a <literal>SpanDuration</literal> is configured for a recording, the recording is active, and the duration of a span of this recording reaches the configured <literal>SpanDuration</literal>, the device shall close all open spans of the recording and open new spans according to the rules of the configured <literal>Format</literal>.
+      </para>
+      <para>
+        All fragments of a span shall be independently processable and presentable with the information of a <literal>MovieBox</literal> in any of the segments of the span.
+        If the information in the <literal>MovieBox</literal> required to process the fragments needs to change while the recording is active, the device shall close all open spans of the recording and start new spans according to the rules of the configured <literal>Format</literal>.
+      </para>
+    </section>
+    <section xml:id="_refCloudRecordingSegments">
+      <title>Segments</title>
+      <para>
+        When a span is opened, the device shall also open a new segment for this span.
+        Inversely, when a span is closed, the device shall close the open segment of this span.
+        When a span is open and the duration of the open segment of this span reaches the <literal>SegmentDuration</literal> configured for the recording, the device shall close the open segment of this span and open a new segment for this span.
+        When closing the open segment of a span and opening a new segment for this span, there shall be no time gap between the two segments.
+      </para>
+      <para>
+        The device shall generate each segment as a fragmented MP4 file according to ISO/IEC 14496-12, the ISO base media file format.
+        A segment shall contain recorded media for the time it was open.
+        The recording formats provide specification about segment types, their contents, and restrictions on the ISO base media file format in the following sections.
+      </para>
+    </section>
+    <section xml:id="_refCloudRecordingSegmentObjects">
+      <title>Segment objects</title>
+      <para>
+        The device shall upload each segment as one object, identified by its object key, to the object storage.
+        The device shall generate the object key for a segment according to the following ABNF <literal>key</literal> rule (according to RFC 5234):
```

Comment:
give an example for the segment object key.

Reply:
I added a section with example keys for segment objects.

Reply:
@fschuetz04, thanks for adding example keys for segment objects.